crates/katana/runner/src/lib.rs

```diff
@@ -29,7 +29,12 @@ impl KatanaRunner {
     }
 
     pub fn new_with_port(port: u16) -> Result<(Self, JsonRpcClient<HttpTransport>)> {
-        let log_filename = format!("logs/katana-{}.log", port);
+        let mut temp_dir = std::env::temp_dir();
+        temp_dir.push("dojo");
+        temp_dir.push("logs");
+        temp_dir.push(format!("{}.log", port));
```

Comment:
```suggestion
        temp_dir.push(format!("katana-{}.log", port));
```
Or do we want a folder dedicated to Katana?

Reply:
for now lets just do `katana-{}`